### PR TITLE
Shapecard display incorrect #104

### PR DESCRIPTION
**Pull request by williamzhu17**

### CODE REVIEW

File: components/utils/ShapeList.js [hunk 1]
```diff
@@ -533,7 +519,7 @@ const ShapeList = ({
                       <ShapeActions className="shape-actions">
                         <ShapeActionsPrimary>
                           <span
-                            onClick={(event, shapeId) =>
+                            onClick={(event) =>
```
Comment (atapas) [inline components/utils/ShapeList.js:522]: Why this change is required in this PR?

Comment (williamzhu17) [reply components/utils/ShapeList.js:522]: it is not required. I just saw some unused variables and wanted to get rid of it

File: components/utils/ShapeList.js [hunk 2]
```diff
@@ -290,19 +290,6 @@ const ShapeList = ({
     setFilteredShape(filterShape(copy, searchTerm));
   }, [searchTerm, shapes, sort]);
 
-  const handleSwicth = (shapeName) => {
```
Comment (atapas) [inline components/utils/ShapeList.js:293]: We don;t need this?

Comment (williamzhu17) [reply components/utils/ShapeList.js:293]: Yes, it was an unused function that I missed.

Comment (atapas) [reply components/utils/ShapeList.js:293]: Ok @williamzhu17 , could you please test out the features one and conform all good to merge?

Comment (williamzhu17) [reply components/utils/ShapeList.js:293]: @atapas just tested all the features and they are all working. should be ready to merge
